src/com/magento/idea/magento2plugin/magento/files/GraphQLSchema.java

```diff
@@ -2,8 +2,35 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 package com.magento.idea.magento2plugin.magento.files;
 
-public class GraphQLSchema {
-    public static String FILE_NAME = "schema.graphqls";
+import com.intellij.lang.Language;
+import com.jetbrains.php.lang.PhpLanguage;
+
+public class GraphQLSchema implements ModuleFileInterface {
```

Comment:
All file models should have suffix **File**

Reply:
fixed